api/src/functions/express/app.ts

```diff
@@ -279,6 +278,8 @@ const airtableUserTestingClient = AirtableUserTestingClient(
 const USERS_TABLE = process.env.USERS_TABLE || "users-table-local";
 const dynamoDb = createDynamoDbClient(IS_OFFLINE, IS_DOCKER, DYNAMO_OFFLINE_PORT);
 const userDataClient = DynamoUserDataClient(dynamoDb, USERS_TABLE, dataLogger);
+const businessesDataClient = DynamoBusinessesDataClient(dynamoDb, BUSINESSES_TABLE, dataLogger);
+const unifiedDataClient = UserAndBusinessSyncClient(userDataClient, businessesDataClient, dataLogger);
```

Comment:
[dust] If we do update the type definition to `DynamoDataClient` I'd like to change this name throughout, too
```suggestion
const dynamoDataClient = UserAndBusinessSyncClient(userDataClient, businessesDataClient, dataLogger);
```